lib/components/atoms/Popover/types/index.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+import type { Node } from 'react';
+
+export type Props = { popOverBody: String, popOverHeader: String, children: Node };
```

Comment:
This is not how we define type String --> string 
I would recommend you change 
popOverHeader --> heading (It should be both string and Node type)
popOverBody --> trigger (This should be a Node or String that will trigger popover)
children --> it should be the content or body of the popover